tools/cldr-code/src/main/java/org/unicode/cldr/util/personname/PersonNameFormatter.java

```diff
@@ -844,7 +923,8 @@ public ParameterMatcher slim() {
 
         public NamePattern getBestMatch(NameObject nameObject, FormatParameters nameFormatParameters) {
             if (nameFormatParameters.order == null) {
-                nameFormatParameters = nameFormatParameters.setOrder(localeToOrder.get(nameObject.getNameLocale()));
+                final Order mappedOrder = localeToOrder.get(nameObject.getNameLocale());
+                nameFormatParameters = nameFormatParameters.setOrder(mappedOrder == null ? Order.givenFirst : mappedOrder);
```

Comment:
Hmm, I wonder whether default to surnameFirst is better. We can discuss in meeting.

Reply:
Will add to agenda

Reply:
added.